Lib/re/sre_compile.py

```diff
@@ -0,0 +1,800 @@
+#
+# Secret Labs' Regular Expression Engine
+#
+# convert template to internal format
+#
+# Copyright (c) 1997-2001 by Secret Labs AB.  All rights reserved.
+#
+# See the sre.py file for information on usage and redistribution.
```

Comment:
```suggestion
# See the re.py file for information on usage and redistribution.
```

`Lib/sre.py` was renamed to `Lib/re.py` in 94a9c09e109706af64ae8796882baab2af25be2f.